wolf-data/src/Wolf/Data/Export/Types.hs

```diff
@@ -89,3 +105,48 @@ instance ToJSON Repo where
             , "notes" .= repoNotes
             , "suggestions" .= repoSuggestions
             ]
+
+type Warn = [Problem]
+
+data Problem
+    = WarnMissingNoteIndex PersonUuid
+    | WarnMissingNote NoteUuid
+    | WarnMissingRelevantPerson PersonUuid
+                                NoteUuid
+    | WarnMissingRelevantNote NoteUuid
+                              PersonUuid
+    deriving (Show, Eq, Generic)
+
+instance ToJSON Problem
+
+instance FromJSON Problem
+
+data Err =
+    NoInitFile
+    deriving (Show, Eq, Generic)
+
+instance ToJSON Err
+
+instance FromJSON Err
+
+prettyPrintErr :: Err -> String
+prettyPrintErr NoInitFile =
+    "Error: the wolf repository is not or poorly initialised."
+
+prettyPrintProblem :: Problem -> String
+prettyPrintProblem (WarnMissingNoteIndex pu) =
+    "Warning: " ++ show pu ++ " has no NoteIndex."
+prettyPrintProblem (WarnMissingNote nu) =
+    "Warning: " ++ show nu ++ " has no note."
+prettyPrintProblem (WarnMissingRelevantPerson pu nu) =
+    "Warning: The note " ++
+    show nu ++ " should mention the person " ++ show pu ++ "."
+prettyPrintProblem (WarnMissingRelevantNote nu pu) =
+    "Warning: The person " ++
+    show pu ++ " should mention the note " ++ show nu ++ " in his noteIndex."
```

Comment:
Instead of writing `X should Y`, which sounds like an opinion.
Write `X does not Y` (while that's what we expected).

Reply:
I changed it to your pleasing.
However, I would like to know why I need to do this.
"Should" means (in my opinion) "it would be better if it did, but it's not absolutely necessary". This, imo, expresses exactly what a "warning" means.